multichain-testing/test/support.ts

```diff
@@ -79,6 +81,66 @@ export const commonSetup = async (t: ExecutionContext) => {
   const hermes = makeHermes(childProcess);
   const nobleTools = makeNobleTools(childProcess);
 
+  const makeCommonAssetInfo = () => {
```

Comment:
🤮 should we export `assetOn` and use it here? 

In some future, this can all be inferred from the registry endpoint and scripted better.